### PR TITLE
fix: Login page doesn't load for new sites

### PR DESCRIPTION
**Pull request by akurungadam**
Login page broken as active domains are not set for new sites

### CODE REVIEW

File: erpnext/templates/includes/footer/footer_powered.html [hunk 1]
```diff
@@ -12,8 +12,8 @@
 } %}
 
 {% set link = '' %}
-{% set label = domains[0].domain %}
```
Comment (nabinhait) [inline erpnext/templates/includes/footer/footer_powered.html:15]: `let label = null;`

Otherwise, it will give error on line 20

**Review by nabinhait — APPROVED** [review]

File: erpnext/templates/includes/footer/footer_powered.html (hunk 1 above)
Comment (nabinhait) [inline erpnext/templates/includes/footer/footer_powered.html:15]: let label = null;

Otherwise, it will give error on line 20

**Review by nabinhait — CHANGES_REQUESTED** [review]

**Comment by Anurag810** [comment]
@nabinhait #22394. both PR are for same issue.

**Review by nabinhait — APPROVED** [review]

**Comment by revant** [comment]
https://travis-ci.com/github/frappe/frappe_docker/jobs/353812594#L827

Added a test to check index page of new site created using develop branch

**Pull request merged**